code/modules/antagonists/__antagonist.dm

```diff
@@ -199,11 +199,9 @@ ABSTRACT_TYPE(/datum/antagonist)
 			. += "<b>[owner.current]</b> (played by <b>[owner.displayed_key]</b>) was \a [assigned_by + display_name]!"
 		else
 			. += "<b>[owner.displayed_key]</b> (character destroyed) was \a [assigned_by + display_name]!"
-		if (length(owner.objectives))
+		if (length(src.objectives))
 			var/obj_count = 1
-			for (var/datum/objective/objective as anything in owner.objectives)
```

Comment:
Reasoning for this? It'll mean any admin given objectives won't be displayed.

Reply:
I'm just discovering that this does't fix the bug it was meant to in the first place, that being that antagonists with no assigned objectives receive the Mission Complete medal.